docs/dictionary/function/mcEncrypt.lcdoc

```diff
@@ -13,6 +13,8 @@ OS: mac, windows, linux
 
 Platforms: desktop, server
 
+Description:
+<reserved word|Reserved> for internal use.
```

Comment:
@seaniepie `mcEncrypt` is interesting because it was previously used for the encrypted result from `ask password` but answer password now returns a clear text result. This means to get legacy behavior you *need* to use `mcEncrypt`. There's an example in `ask password`. I would say noting that here would be helpful but also recommending using the encrypt command instead.